src/main/java/com/crowdin/client/translations/model/PreTranslationReportResponse.java

```diff
@@ -0,0 +1,31 @@
+package com.crowdin.client.translations.model;
+
+import lombok.Data;
+import java.util.List;
+import java.util.Map;
+@Data
+public class PreTranslationReportResponse {
+
+    private List<Language> languages;
+    private String preTranslateType;
+
+    @Data
+    public static class Language {
```

Comment:
Use the `TargetLanguage` naming convention to align with the terminology in the API documentation